contracts/extensions/ERC721AOwnersExplicit.sol

```diff
@@ -13,18 +13,21 @@ abstract contract ERC721AOwnersExplicit is ERC721A {
      */
     function _setOwnersExplicit(uint256 quantity) internal {
         require(quantity != 0, 'quantity must be nonzero');
-        require(currentIndex != 0, 'no tokens minted yet');
+        require(_nextTokenId > 1, 'no tokens minted yet');
+        if (nextOwnerToExplicitlySet == 0) {
```

Comment:
Should check and set the _nextOwnerToExplicitlySet local variable instead for gas savings. 

See https://github.com/chiru-labs/ERC721A/pull/59